README.md

```diff
@@ -40,3 +81,4 @@ There are a new set of labels available project wide:
 
  * A PR that signals the completion of an important Epic worthy of making some noise about and including in a newsletter can be labeled `area/community`, `kind/news`, and `kind/marketing`.
  * An issue is related to the experience of users joining the project to become contributors is labeled `area/community`, `area/user`, `area/contributor`, `kind/experience`, and `kind/onboarding`.
+ * A write-up on how to setup a clone of a repository for doing work intended to be contributed to the community would be labeled `area/contributor`, `kind/documentation`, `kind/onboarding`, and `kind/handbook`.   
```

Comment:
@quaid the lint is failing here, you need to remove the trailing white space after this line, and also add a newline (with no whitespace). 

If you run `pre-commit run --all-files` from the root of this commit, it should automatically fix these errors for you

Reply:
Oh! This is unhappy with the other PR, that's part of the mystery to me.

I'm going to write out my understanding to make sure I'm not missing something:
* If I forget to run `pre-commit install` on a new repo I've cloned, it won't run `pre-commit` locally and I will get two failed checks—one for not having a pre-commit environment running, the other for any failures of the actual commit itself in check.
* If I then run `pre-commit install`, the only way the bit knows and can test if I have done that is if I commit any change to the repo so it shows the change to my local installation. Is that accurate?
* If there are any files that fail pre-commit normally, I get to fix them before the commit. If I commit files without pre-commit and there _are_ errors in the files, then running pre-commit subsequently will catch and fix those files. I now have actual changes to those files, so restaging and commiting them is what I would do anyway.

Thanks!